src/commands/features/create.ts

```diff
@@ -0,0 +1,58 @@
+import { createFeature, CreateFeatureParams } from '../../api/features'
+import { descriptionPrompt, keyPrompt, namePrompt, sdkVisibilityPrompt } from '../../ui/prompts'
+import CreateCommand from '../createCommand'
+import { VariableListOptions } from '../../ui/prompts/listPrompts/variablesListPrompt'
+import { Flags } from '@oclif/core'
+
+export default class CreateFeature extends CreateCommand {
+    static hidden = false
+    authRequired = true
+    static description = 'Create a new Feature.'
+    static flags = {
+        ...CreateCommand.flags,
+        variables: Flags.string({
+            description: 'The variables to create for the feature'
+        }),
+        sdkVisibility: Flags.string({
+            description: 'The visibility of the feature for the SDKs'
+        })
+    }
+
+    prompts = [keyPrompt, namePrompt, descriptionPrompt]
+
+    public async run(): Promise<void> {
+        const { flags } = await this.parse(CreateFeature)
+        const { headless, key, name, description, variables, variations, sdkVisibility } = flags
+        await this.requireProject()
+
+        if (headless && (!key || !name)) {
+            this.writer.showError('In headless mode, the key and name flags are required')
+            return
+        }
+
+        const params = await this.populateParameters(CreateFeatureParams, this.prompts, {
+            key,
+            name,
+            description,
+            variables: variables ? JSON.parse(variables) : [],
```

Comment:
What happens if the variables/variations string is malformed? Does this just throw an exception?

Reply:
i haven't committed it yet but I'm gonna wrap most of this functionality in a try catch so we can properly catch errors

Reply:
ahh okay - nice!